server/routes/activitiesMigration/activitiesMigrationController.ts

```diff
@@ -102,13 +104,25 @@ export default class ActivitiesMigrationController {
         })
         return 'Error checking ACTIVITY feature switch'
       }),
-    ]).then(([estimatedCount, dlqCount, incentiveLevels, featureSwitchOn]) => {
+
+      this.activitiesService.getRolloutPrison(prisonId, context(res)).catch(error => {
+        errors.push({
+          text: `Failed to check if prison ${prisonId} is switched on in DPS due to error: ${error.message}`,
+          href: '',
+        })
+        return null
+      }),
+    ]).then(([estimatedCount, dlqCount, incentiveLevels, nomisFeatureSwitchOn, dpsPrisonRollout]) => {
       req.session.startActivitiesMigrationForm.estimatedCount = estimatedCount.toLocaleString()
       req.session.startActivitiesMigrationForm.dlqCount = dlqCount.toLocaleString()
       req.session.startActivitiesMigrationForm.incentiveLevelIds = incentiveLevels.map(
         (level: IncentiveLevel) => level.code,
       )
-      req.session.startActivitiesMigrationForm.prisonSwitchedOn = featureSwitchOn
+      req.session.startActivitiesMigrationForm.prisonSwitchedOnNomis = nomisFeatureSwitchOn
+      req.session.startActivitiesMigrationForm.prisonSwitchedOnDps =
+        dpsPrisonRollout === null ||
+        (dpsPrisonRollout.activitiesRolledOut &&
+          dpsPrisonRollout.activitiesRolloutDate <= moment().format('YYYY-MM-DD'))
```

Comment:
This is the main change - get the `PrisonRolloutPlan` for the prison and check if the prison is rolled out on or before today.